docs/man/plastic.1

```diff
@@ -0,0 +1,90 @@
+PLASTIC(1)      General Commands Manual      PLASTIC(1)
+
+NAME
+
+    plastic - NES emulator with graphical (GUI) and terminal (TUI) interfaces.
+
+SYNOPSIS
+
+    plastic [rom-file]
+    plastic_tui [rom-file] [-a]
+
+DESCRIPTION
+
+    plastic is an NES emulator written in Rust, designed to run NES games with both a graphical (GUI) and a terminal-based (TUI) interface.
+
+    The graphical interface (plastic) can be launched without any arguments (though it can be started from the terminal with a ROM file) and provides a user-friendly graphical experience.
+
+    The terminal-based interface (plastic_tui) allows users to run the emulator in the terminal with optional flags for additional control.
+
+    Saved states are stored at the following path:
+    ~/.local/share/plastic/saved_states
+
+OPTIONS
+
+    plastic
+        Launches the emulator with the graphical interface. This mode does not require arguments but can accept them if needed.
```

Comment:
This can also be modified.

Reply:
It states that it doesnt need it but if you provide them it will follow the arguments. And thats acurate. The thing is most people dont launch GUIs from a Terminal and rather use an Applauncher thats why i made a .desktop file and put it in the debian package.  But if you insist.